File: 3483-unique-3-digit-even-numbers/3483-unique-3-digit-even-numbers.js
function totalNumbers(digits) {
    const s = new Set();
    for (let i = 0, n = digits.length; i < n; i++) {
        for (let j = 0; j < n; j++) {
            for (let k = 0; k < n; k++) {
                if (i !== j && i !== k && j !== k) {
                    s.add(100 * digits[i] + 10 * digits[j] + digits[k]);
                }
            }
        }
    }
    return [...s].filter((v) => v > 99 && !(v % 2)).length;
}

File: 3483-unique-3-digit-even-numbers/3483-unique-3-digit-even-numbers.py
class Solution:
    def totalNumbers(self, digits: List[int]) -> int:
        s, n = set(), len(digits)
        for i in range(n):
            for j in range(n):
                for k in range(n):
                    if i != j and i != k and j != k:
                        s.add(100 * digits[i] + 10 * digits[j] + digits[k])
        return sum(v > 99 and v % 2 == 0 for v in s)